src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,8 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+  const employeeObject = employees.find(
+    (element) => element.name === employeeName
+  );
+  return employeeObject.role;
+};
```

Comment:
This could be simplified into a one liner without losing and clarity:
```suggestion
  return employees.find(element => element.name === employeeName).role;
};
```